set.mm

```diff
@@ -150198,7 +150198,7 @@ because the border cases ( ` M = N ` , ` -. ( M ..^ N ) C_ ( 0 ..^ L ) `
 
   ${
     $d f l n w $.
-    $( Perform a cyclical shift for an arbitrary class.  Meaningful only for
+    $( TODO-AV: USE "prefix"! Perform a cyclical shift for an arbitrary class.  Meaningful only for
```

Comment:
Here and later theorems: if some TODOs are needed, I would put them at the end of comments (and exclamation mark is not needed). This is not in a mathbox, and the comments are intended for users, not contributors.

Reply:
The TODOs regarding this PR are completely removed. I will have a look at the other TODOs marked with TODO-AV later.